src/main/scala/nodes/learning/KernelMatrix.scala

```diff
@@ -25,49 +25,52 @@ trait KernelMatrix {
 
 }
 
-class BlockedKernelMatrix (kernelGen: KernelGenerator, data: RDD[DenseVector[Double]],  train: Boolean, cacheKernel: Boolean) extends KernelMatrix
+class BlockedKernelMatrix (kernelGen: KernelGenerator,
+                           data: RDD[DenseVector[Double]],
+                           train: Boolean,
```

Comment:
Can we detect this `same` automatically by comparing RDD ids ? Or will that not work ?